src/rf/rf/urls.py

```diff
@@ -8,19 +8,13 @@
 from rest_framework import routers
 
 import apps.home.urls
-import apps.home.views
-
+import apps.user.urls
 
 admin.autodiscover()
 
-router = routers.DefaultRouter()
-router.register(r'users', apps.home.views.UserViewSet)
-
 urlpatterns = patterns(
     '',
     url(r'^', include(apps.home.urls)),
-    url(r'^api/', include(router.urls)),
-    url(r'^api-auth/', include('rest_framework.urls',
-                               namespace='rest_framework')),
+    url(r'^auth/', include(apps.user.urls)),
```

Comment:
Ok, I changed it.

On Wed, Aug 26, 2015 at 10:38 AM, Kevin DeLoach <notifications@github.com>
wrote:

> In src/rf/rf/urls.py
> <https://github.com/azavea/raster-foundry/pull/39#discussion_r37989474>:
>
> >  urlpatterns = patterns(
> >      '',
> >      url(r'^', include(apps.home.urls)),
> > -    url(r'^api/', include(router.urls)),
> > -    url(r'^api-auth/', include('rest_framework.urls',
> > -                               namespace='rest_framework')),
> > +    url(r'^auth/', include(apps.user.urls)),
>
> We typically use a URL prefix that matches the name of the app. So this
> should by /user/login etc.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/azavea/raster-foundry/pull/39/files#r37989474>.
>



-- 
Lewis Fishgold, Software Developer
Azavea |  340 N 12th St, Ste 402, Philadelphia, PA
Web azavea.com <http://www.azavea.com/>  |  Blog azavea.com/blogs
<http://www.azavea.com/Blogs>  | Twitter @azavea <http://twitter.com/azavea>